jar/backup.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package jar
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+// backupFile makes copies JARs file in the backup folder
+func backupFile(jarPath string, jarSource string, jarFile string) error {
+	pathdst := filepath.Join(jarPath, "backup")
```

Comment:
As discussed in #13, I think this should just append the suffix, not create a directory. So

/foo/bar/my.jar

Would be renamed to

/foo/bar/my.jar.bak